torch_xla/csrc/ops/eigh.cpp

```diff
@@ -0,0 +1,59 @@
+#include "torch_xla/csrc/ops/eigh.h"
+
+#include <array>
+
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/tensor_util.h"
+#include "torch_xla/csrc/torch_util.h"
+#include "xla/client/lib/self_adjoint_eig.h"
+
+namespace torch_xla {
+
+namespace {
+
+std::array<xla::XlaOp, 2> LowerImpl(xla::XlaOp input, bool lower) {
+  auto [eigenvectors, eigenvalues] =
+      xla::SelfAdjointEig(input, lower, /* max_iter */ 64, /* tol */ 1e-6);
```

Comment:
What are the max_iter and tol (for opaque) used for? 
Also, could you add a comment on why changing the default value?

Reply:
When testing I discovered that the default settings lead to a very low accuracy in the reconstructed matrix (e.g. let's say we decompose `A` to `A' = V @ Q @ V_T`, then `A` and `A'` have a difference in elements above 0.1.

After looking at what JAX does I think it can be simpler to align with JAX: https://github.com/google/jax/blob/a8b425cac50c842f66f36903dfb93fe6ad5a2a5b/jax/_src/lax/linalg.py#L726. Looks like they use the same tolerance but higher max_iter.